src/contrib/cluster/Akka.DistributedData/Serialization/Proto/OtherMessage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Akka.DistributedData.Serialization.Proto.Msg
+{
+    internal sealed partial class OtherMessage : IComparable<OtherMessage>
```

Comment:
Added so that OtherMessage can be sorted based on its content (SHA-1 digest consistency)